Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -109,11 +109,11 @@ def test_bottle
     formula_file = CoreFormulaRepository.new.formula_dir/"testball.rb"
     formula_file.write <<-EOS.undent
       class Testball < Formula
-        url "https://example.com/testabll-0.1.tar.gz"
+        url "https://example.com/testball-0.1.tar.gz"
       end
     EOS
     HOMEBREW_CACHE.cd do
-      assert_match /testball-0\.1.*\.bottle\.tar\.gz/,
+      assert_match %r{testball-0\.1.*\.bottle\.tar\.gz},
```

Comment:
There's very little I dislike more stylistically than parens around `assert_match`es. Think it looks exceptionally messy.

That said, since two of you liked the idea and only I'm against I'll implement it.